packages/lesswrong/components/localGroups/EventTime.tsx

```diff
@@ -47,7 +45,7 @@ const EventTime = ({post, dense=false}: {
   // less sense, but users can enter silly things.)
   else if (!start || !end) {
     const eventTime = start ? start : end;
-    return <>{eventTime!.calendar({}, calendarFormat)}</>
+    return <>{eventTime!.calendar(calendarFormat)} {eventTime!.format('z')}</>
```

Comment:
`calendar()`'s default formats don't include timezone, but since the time is always at the end we can just add our own timezone there